langs/tree-sitter-langs-build.el

```diff
@@ -339,10 +339,9 @@ non-nil."
       ;; FIX: Handle HTTP errors properly.
       (url-copy-file (tree-sitter-langs--bundle-url version os)
                      bundle-file 'ok-if-already-exists)
-      (with-temp-buffer
-        (insert-file-contents bundle-file)
-        (tar-mode)
-        (tar-untar-buffer))
+      (shell-command (format "tar -xvzf %s"
```

Comment:
How about `(tree-sitter-langs--call "tar" "-xvzf" bundle-file)`?

Reply:
Yes, it works!